### ISSUE TITLE
Title and role attributes are missing for functional control  in Admin

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#785]
**Describe the bug**
Title and role attributes are missing for functional control  in Admin screen.

**To Reproduce**
1. Go to Admin (http://localhost:9992/cm/app/?view=workflow&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Click on plus icon given against "Assigned" label in left side. A new pop up will appear.
3. Mouse over on  the minimize and maximize icons, given on the left side of the Asset and Sites label.
4. No title/label displayed.

**Expected behavior**
Title/Label should be displayed on each component of the screen.
